infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespace.java

```diff
@@ -113,14 +113,21 @@ public KubernetesNamespace(
    * <p>Preparing includes creating if needed and waiting for default service account.
    *
    * @param markManaged mark the namespace as managed by Che if it is newly created in this method
+   * @param canCreate true if the namespace can be created during the prepare, false if the
```

Comment:
weird word order in this sentence. Maybe something like "If true, the namespace will be created when does not exist. If false, namespace must already exist otherwise exception will be thrown." ?